### PR TITLE
Cpp rmsd

### PR DESCRIPTION
**Pull request by iparask**
The MPI c++ code should be in synch with [this code](https://github.com/radical-experiments/midas_exps/blob/master/RMSD-MDAnalysis/mpi4py/randomArray.py)

I run it once and I saw ranks that took 0ms, which is not true. I think I have some bug in the way I am getting timestamps or gathering the timing results.

### CODE REVIEW

**Comment by iparask** [comment]
The request is for the rmsd.cpp file which is the C++ equivalent - I think - of the python file linked above.

Do not review any file that is in the ```includes``` folder

File: RMSD/rmsd.cpp [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    ierr = MPI_Init ( NULL, NULL );
+//
+//  Get the number of processes.
+//
+    ierr = MPI_Comm_size ( MPI_COMM_WORLD, &size );
+//
+//  Get the individual process ID.
+//
+    ierr = MPI_Comm_rank ( MPI_COMM_WORLD, &rank );
+
```
Comment (andre-merzky) [inline RMSD/rmsd.cpp]: You add three lines of comments to the most basic MPI methods, which are not really needed - but don't check error codes... :P

File: RMSD/rmsd.cpp [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    struct timeval tp; //Time struct.
+
+
+
+    int *duration = new int[3]; //keeps track of durations for each process
+    int *durations = new int[size*3]; // Pointer that will gather all the information
+
+    // Get timestamp and convert it to ms
+    gettimeofday(&tp, NULL);
+    tstart = tp.tv_sec * 1000 + tp.tv_usec / 1000;
```
Comment (vivek-bala) [inline RMSD/rmsd.cpp]: I will suggest creating a function to get the current time and then using it. Something like:
```
float current_time(){
    gettimeofday(&tp, NULL);
    current_time = tp.tv_sec * 1000 + tp.tv_usec / 1000;
    return current_time;
}
```

File: RMSD/rmsd.cpp (hunk 1 above)
Comment (iparask) [reply RMSD/rmsd.cpp]: That is because of a CP from an example :P It needs to be removed. You are right!

File: RMSD/rmsd.cpp (hunk 2 above)
Comment (iparask) [reply RMSD/rmsd.cpp]: Okay! I think you are right!

Comment (vivek-bala) [reply RMSD/rmsd.cpp]: Please don't judge my c skills. That was not the point of this comment! :P

**Pull request merged**